pom.xml

```diff
@@ -397,6 +398,20 @@
                 <version>${log4j.version}</version>
             </dependency>
             <dependency>
+              <!-- This is not slf4j's version because of performance concerns
+              http://www.slf4j.org/api/org/slf4j/bridge/SLF4JBridgeHandler.html
+              Please make sure to do performance tests before switching this to slf4j
+              -->
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-jul</artifactId>
+                <version>${log4j.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>jcl-over-slf4j</artifactId>
+                <version>${slf4j.version}</version>
```

Comment:
Alternatively, you could also use log4j2's own jcl bridge: http://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-jcl
http://logging.apache.org/log4j/log4j-2.1/log4j-jcl/index.html
Since druid prefers to uses log4j2 config, then it might be better performance wise to use their bridges vs going from jcl->slf4j->log4j for example. Same could be said for the comment above about slf4j jul bridge. But I may be missing something in the underpinnings of log setup :)